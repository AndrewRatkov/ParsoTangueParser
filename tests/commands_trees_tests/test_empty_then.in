int x := 15;
int y := 90;
if x > y then
else
x := y;
fi