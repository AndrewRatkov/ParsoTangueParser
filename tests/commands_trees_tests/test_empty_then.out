[DEF int x]--->[INT,15]
[DEF int y]--->[INT,90]
IF
|
+--->[INT,> ]------->[INT,x]
|        |
|        +---------->[INT,y]
|
THEN
|
ELSE
|
+--->[DEF int x]--->[INT,y]
|
FI
